clients/src/main/java/io/strimzi/http/producer/HttpProducerClient.java

```diff
@@ -104,11 +113,21 @@ public void checkAndSendMessages() {
     }
 
     public ProducerRecord generateMessage(int numOfMessage) {
-        String record = "{\"records\":[{\"key\":\"key-" + numOfMessage + "\",\"value\":\"" + configuration.getMessage() + "-" + numOfMessage + "\"}]}";
+        String message;
+        if (this.configuration.getMessageTemplate() != null) {
+            message = dataGenerator.generateStringData();
+        } else {
+            message = configuration.getMessage() + "-" + numOfMessage;
+        }
+        if (this.configuration.getMessageType().equalsIgnoreCase(MessageType.TEXT.name())) {
+            message = "\"" + message + "\"";
+        }
```

Comment:
I think it's not needed - the double quotes were needed because of the bridge clients, where we expected JSON as a message type, even though we send text type message -> so it was parsed.
When the double quotes were deleted, we send the message `Hello world - X` without the quotes in the text type, but it will parse just the `Hello` in the JSON type, causing some issues on the Strimzi bridge side.